concourse/tasks/run-task.sh

```diff
@@ -31,26 +31,21 @@ echo "Starting task..."
 task=$(aws ecs run-task --cluster $CLUSTER \
 --task-definition $task_definition_arn --launch-type FARGATE --count 1 \
 --network-configuration $network_config \
+--started-by "Concourse" \
 --overrides '{
   "containerOverrides": [{
     "name": "'"$APPLICATION"'",
     "command": ["/bin/bash", "-c", "'"$COMMAND"'"]
   }]
 }')
 
-task_arn=$(echo $task | jq .tasks[0].taskArn)
-
+task_arn=$(echo $task | jq .tasks[0].taskArn -r)
+task_id=$(basename $task_arn)
 echo "waiting for task $task_arn to finish..."
-
-aws ecs wait tasks-stopped --tasks=[$task_arn] --cluster $CLUSTER
-
+aws ecs wait tasks-stopped --tasks="$task_id" --cluster $CLUSTER
```

Comment:
style: will it be still valid that we use space as separator, --tasks="$task_id" -> --tasks "$task_id" ? Just to keep consistency.

Reply:
Thanks, I've removed the `=` from them 👍